<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormatDrivesDialog</class>
 <widget class="QDialog" name="FormatDrivesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Format drives</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Format drives</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Available drives</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="refreshListButton">
     <property name="text">
      <string>Refresh list</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Label: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newLabel">
       <property name="toolTip">
        <string>Maximum label size is 9 characters.</string>
       </property>
       <property name="maxLength">
        <number>9</number>
       </property>
       <property name="placeholderText">
        <string>OPTIONAL: Type the label you want on your drives after formating.</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="drivesList">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="log">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="showAllDrives">
       <property name="text">
        <string>Show all drives</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="formatButton">
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close window</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
